languages/julia/exercises/concept/methods.basic/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
```

Comment:
There are currently no hints specified in the `hints.md` for any of the tasks here. Was that intentional?

Reply:
Somewhat. I wasn't sure what should go in there. I think `introduction.md` contains enough information that you can easily solve the exercise without a hint. I'm not sure what kind of hint would be helpful that wouldn't either be a duplicate of `introduction.md`, or, if it'd link to the manual like in the C#/F# implementation for example, go into way too much detail. Do you have any ideas on hints that would work well here?

Reply:
Well no. At this point, I just use very [generic hints](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/basics/.docs/hints.md), like re-iterating things like "You can use the mathematical operator for subtraction to subtract values.". 

I also link to resources that can be useful. That said, I don't know if Julia has such resources, or if the manual is just a big listing.

Reply:
It has a manual that would work, but sometimes it goes into way too much detail and, imo, fits `after.md` better than the hints because it could be quite overwhelming. I guess I can add links to the [function section](https://docs.julialang.org/en/v1/manual/functions/) and specify "how far" to read :D

Reply:
That could work. I found that section nice to read.

Reply:
I've added a generic hint. Repeating it for each step seems like overkill since they'd just link to the same page anyway.